test/conftest.py

```diff
@@ -0,0 +1,69 @@
+from pytest import fixture
+import reciprocalspaceship as rs
+import numpy as np
+import gemmi
+from meteor.utils import canonicalize_amplitudes
+
+
+@fixture
+def random_intensities() -> rs.DataSet:
+    """
+    A simple 10x10x10 P1 dataset, with random intensities
+    """
+
+    params = (10.0, 10.0, 10.0, 90.0, 90.0, 90.0)
```

Comment:
This is a lot of copypasta, please reduce

Reply:
good point, done